h2o-bindings/bin/gen_R.py

```diff
@@ -198,8 +198,8 @@ def gen_module(schema, algo, module):
         yield "    beta_constraints <- as.h2o(beta_constraints)"
         yield "  }"
     elif algo == "generic":
-        yield "  # Required args: model_key"
-        yield "  if (is.null(model_key)) stop(\"argument 'model_key' must be provided\")"
+        yield "  # Required args: either model_key or path"
+        yield "  if (is.null(model_key) && is.null(path)) stop(\"argument 'model_key' or 'path' must be provided\")"
```

Comment:
should the same be done for py?

Reply:
I'm doing that for all the algos in the very next PR. Will definitely be there.

Reply:
Python is more permissive and only checks against list of allowed algos, thus no hotfix is needed, but the check will be unified. There is a backend check as well btw.

Reply:
I'll backport the change here.